main.py

```diff
@@ -0,0 +1,154 @@
+import os.path
```

Comment:
please, remove files that are not related to labs implementation

Reply:
done